lib/pages/classes/service/class_provider.dart

```diff
@@ -287,4 +288,14 @@ class ClassProvider with ChangeNotifier {
       return false;
     }
   }
+
+  Future<RemoteConfigService> getRemoteConfig() async {
+    try {
+      final remoteConfig = await RemoteConfigService.getInstance();
+      return remoteConfig;
+    } catch (e) {
+      AppToast.show(S.current.errorSomethingWentWrong);
+      return null;
+    }
+  }
```

Comment:
why is this in classProvider? shouldn't it be some globally accessible variable and initialized at startup? Sure it's only used for classes now, but it doesn't mean we won't use it for other things. Make it a global variable in the utils.dart file maybe, and fetch it in `_setUpAndChooseStartScreen`.